xml/System/GCMemoryInfo.xml

```diff
@@ -75,9 +75,9 @@ A GC is identified by its <xref:System.GCMemoryInfo.Index>, which starts from 1
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a value that indicates if this is a concurrent GC (BGC) or not.</summary>
+        <summary>Gets a value that indicates if this is a background or concurrent GC or not.</summary>
```

Comment:
background GCs are concurrent. so I would keep the original text; or if you want to make it more explicit you could say 

Gets a value that indicates if this is a background (which is concurrent) or not.